LondonBritishLibrary/orient/BLorient6466.xml

```diff
@@ -0,0 +1,427 @@
+                                <locus target="#145v"/>
+                                <desc>A drawing in black ink representing the owner seeking protection of <persName ref="PRS9162TaklaHa"/>.
+                                    </desc>
+                            </decoNote>   
+                        </decoDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="e3">
+                                    <locus target="#145r #146r"/>
+                                    <desc type="OwnershipNote">The name of the owner is <persName role="owner" ref="PRS14359GabraSellase"/>.
+                                        </desc>
+                                </item>
+                                <item xml:id="e4">
+                                    <locus target="#170ra #171ra #173ra #175ra #177rb #179ra #180rb"/>
+                                    <desc type="findingAid">The text is divided into the days of the week.</desc>
+                                </item>
+                            </list>
+                        </additions>
+                    </physDesc>
+                        <additional>
+                            <adminInfo>
+                                <recordHist>
+                                    <source>
+                                        <listBibl type="catalogue">
+                                            <bibl>
+                                                <ptr target="bm:Strelcyn1978BritishLibrary"/>
+                                                <citedRange unit="page">26-27</citedRange>
+                                            </bibl> 
+                                        </listBibl>
+                                    </source>
+                                </recordHist>
+                            </adminInfo>
+                        </additional>
+                    </msPart>
+                    <msPart xml:id="p4">
+                        <msIdentifier>
+                            <idno/>
+                        </msIdentifier>
+                        <msContents>
+                            <msItem xml:id="p4_i1">
+                                <locus from="ii" to="iii"/>
+                                <title ref="LIT2248salota"/>
+                                <note>Written on the verso of the first end flyleaf and on the recto and verso of the second end flyleaf.</note>
+                                <incipit xml:lang="gez">ነአኵቶ፡ ለገባሬ፡ ሠናያት፡ በላዕሌነ፡ ለእግዚአብሔር፡ መሐሪ፡ አቡሁ፡ ለእግዚእነ፡ ወአምላክነ፡ ወመድኃኒነ፡ ኢያሱስ፡ ክርስቶስ፡</incipit>
+                            </msItem>
+                        </msContents>
+                        <physDesc>
+                            <objectDesc form="Codex">
+                                <supportDesc>
+                                    <support>
+                                        <material key="paper">Oriental paper</material>
```

Comment:
Wow. Nothing in the catalogue on the provenance?

Reply:
Do you mean the provenance of the paper, of the codicological units or of the manuscript as a whole?

Reply:
the entire manuscript :)

Reply:
There is nothing said on the provenance of this manuscript, except for the note, that it was bought from PRS14360Owen, whom I encoded in e5. Do you think it is reasonable to record this transaction in `<provenance>` as well?

Reply:
In general yes, I think it is useful to briefly summarize the information on the manuscript's history in the dedicated elements.